docs/pages/web/datepicker.js

```diff
@@ -221,7 +221,7 @@ export default function DatePickerPage({ generatedDocGen }: {| generatedDocGen:
           description="DatePicker supports disabling future & past dates as well as an array of selected dates."
         >
           <MainSection.Card
-            title="Disable future & past"
+            title="Disable past & future dates"
```

Comment:
Heads up, noticing these weird extra headers:
<img width="392" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/87a967ec-12d7-45cf-a5e3-d46760034188">

If we use a bold header/title like this, can you add a description underneath

Reply:
DONE